### DISCUSSION TITLE
Hydra Login Accept when Kratos session

### DISCUSSION THREAD

**Discussion by MiraBlack** [ory/hydra#3500] [Q&A]
As I suppose the hydra skip allows you to skip login if this client already passed checks and received a token. I have an integration with kratos and I want hydra  acceptLoginRequest if there is a current kratos session. In the examples, the admin api of Hydra is called on the login page for this. It is not safe.  Are there any other options?  Is it possible to change the skip flag after kratos check for a session (using toSession on front)?  Is it possible to add such logic to Kratos? Thanks!

**Comment by vinckr** [comment]
Hey @MiraBlack 

The skip flag in Hydra is used to indicate whether the client has previously requested the same scopes from the same user. If skip is true, you can bypass asking the user to grant the requested scopes and simply forward the user to the redirect URL. This feature allows you to update or set session information.
However, the documentation does not provide specific information on how to change the skip flag after Ory Kratos checks for a session or how to add such logic to Ory Kratos.
It's important to note that any changes to the skip flag or the session management process should be done carefully to maintain the security and integrity of the authentication process

Let me know if you find a solution or if you hear something else.
